frontend/src/SeesawApp.vue

```diff
@@ -275,6 +276,9 @@ export default defineComponent({
             },
     mounted (){
         console.log('Vue App object avail in window.VueApp');
+        console.log("GOOD LUCK AGAIN"); 
+        console.log("10th time's the charm buddy"); 
+        console.log("Another one"); 
```

Comment:
Maybe we should remove console log statements now that you're done debugging it (we can leave one if needed).